### ISSUE TITLE
Doesn't allow use of the word "Of" in type names

### ISSUE THREAD

**Issue by iantabron** [fgilde/MakeGenericAgain#3]
Replace of the word "Of" is too eager. If any 'regular' type names contain the word "Of" they are made generic. For example: `EndOfLineAllocation` type becomes `End<LineAllocation>` incorrectly.

I believe the only reasonable resolution to this issue would be to accept a blacklist of type names to ignore as options

**Comment by iantabron** [comment]
#4 resolves this issue

**Comment by Daaaaemon** [comment]
The -i argument doesn't work, the tool fails to parse the input argument:
```
dotnet makeGenericAgain -f "Client.cs" -i "UnitOfMeasure"
Unhandled exception. System.ArgumentException: Object of type 'System.String' cannot be converted to type 'System.Collections.Generic.ICollection`1[System.String]'.
   at System.RuntimeType.CheckValue(Object& value, Binder binder, CultureInfo culture, BindingFlags invokeAttr)
   at System.Reflection.RuntimePropertyInfo.SetValue(Object obj, Object value, Object[] index)
   at MakeGenericAgain.CommandLineArgs.Parse[T](String args) in C:\dev\privat\github\MakeGenericAgain\MakeGenericAgain\CommandLineArgs.cs:line 38
   at MakeGenericAgain.CommandLineArgs.Parse[T](String[] args) in C:\dev\privat\github\MakeGenericAgain\MakeGenericAgain\CommandLineArgs.cs:line 12
   at MakeGenericAgain.Program.Main(String[] args) in C:\dev\privat\github\MakeGenericAgain\MakeGenericAgain\Program.cs:line 13```

**Comment by fgilde** [comment]
Will have a look at it and fix it

**Comment by fgilde** [comment]
Its fixed in 1.0.6 https://www.nuget.org/packages/MakeGenericAgain/1.0.6
